ArjunaCore/arjuna/classes/com/arjuna/ats/internal/arjuna/recovery/PeriodicRecovery.java

```diff
@@ -787,10 +845,24 @@ private void doWorkInternal()
             ClassLoader cl = switchClassLoader(m);
             try {
             m.periodicWorkSecondPass();
+            if (m instanceof SuspendBlockingRecoveryModule) {
+                /*
+                 * Once Recovery Modules (implementing SuspendBlockingRecoveryModule) indicate
+                 * they do not want to block recovery (i.e. there is more work to do), they will
```

Comment:
I think "there is more work to do" should be "there is no more work to do"

Reply:
Missed the "no" :face_with_peeking_eye: . Thanks Tom